content/en/docs/contribute/style/content-guide.md

```diff
@@ -45,68 +45,68 @@ effort from project maintainers and tends to become outdated more quickly.
+    1. If no, then:
+        1. Stop!
+            1. Adding content about vendor-specific products is not allowed
+            1. Linking to vendor-specific documentation and websites is not allowed
 
 ### What is and isn't allowed
 
 There are some scenarios in which the Kubernetes documentation includes content from non-Kubernetes projects. 
 Below are general categories of non-Kubernetes project content along with guidelines of what is and is not allowed:
 
 1. Instructional content involving non-Kubernetes projects during setup or operation of Kubernetes
-    - Allowed:
-        - Referring to or linking to existing documentation about a CNCF project or a project in the kubernetes or kubernetes-sigs GitHub organizations
-            - Example: for installating Kubernetes in a learning environment, including a prerequisite stating that successful installation and configuration of minikube is required and linking to the relevant minikube documentation
-        - Adding content for kubernetes or kubernetes-sigs projects that don't have their own instructional content
-            - Example: including [kubadm](https://github.com/kubernetes/kubeadm) installation and troubleshooting instructions    
-    - Not Allowed:
-        - Adding content that duplicates documentation in another repository
-            - Examples: 
-                - Including minikube installation and configuration instructions; minikube has its own [documentation](https://minikube.sigs.k8s.io/docs/) that provides those instructions
-                - Including instructions for installing Docker, CRI-O, containerd, and other container runtimes on various operating systems
-                - Including instructions for installing Kubernetes on production environments using various projects:
-                    - Kubernetes Rebar Integrated Bootstrap (KRIB) is a vendor-specific project and content belongs in the vendor's documentation
-                    - [Kubernetes Operations (kops)](https://github.com/kubernetes/kops) has installation instructions and tutorials in its GitHub repository
-                    - [Kubespray](https://kubespray.io) has its own documenation  
-        - Adding a tutorial that explains how to perform a task using a vendor-specific product or an open source project that is not a CNCF project or a project in the kubernetes or kubnetes-sigs GitHub organizations
-        - Adding a tutorial on how to use a CNCF project or a project in the kubernetes or kubnetes-sigs GitHub organizations if the project has its own documentation
+    1. Allowed:
+        1. Referring to or linking to existing documentation about a CNCF project or a project in the kubernetes or kubernetes-sigs GitHub organizations
+            1. Example: for installating Kubernetes in a learning environment, including a prerequisite stating that successful installation and configuration of minikube is required and linking to the relevant minikube documentation
```

Comment:
nit: line 65, spelling fix, `installating` 😃
nit: line 54, start as `Installing Kubernetes ...` or list the three items (?), add a comma after required?

Reply:
This will be done in a separate PR.  I've reverted my content-guide.md changes.